### PR TITLE
Jonas python ii

### PR DESCRIPTION
**Pull request by UnknownMonk**

### CODE REVIEW

File: src/adv.py [hunk 1]
```diff
@@ -19,6 +20,8 @@
     'treasure': Room("Treasure Chamber", """You've found the long-lost treasure
 chamber! Sadly, it has already been completely emptied by
 earlier adventurers. The only exit is to the south."""),
+
+    'boomboom_room': Room("BoomBoom Room", """You've found the long-lost boom boom room! Get the chanpagin ready the ladies are on the way.Why would you want to leave. The only exit is to the south."""),
```
Comment (tryingtokeepup) [inline src/adv.py]: Boom boom in the zoom zoom.

File: src/adv.py [hunk 2]
```diff
@@ -38,14 +41,37 @@
 #
 
 # Make a new player object that is currently in the 'outside' room.
+current_player = Player("chuckD", room['outside'])
```
Comment (tryingtokeepup) [inline src/adv.py]: Good job there on figuring out how to send those arguments to the Player class!

File: src/adv.py [hunk 3]
```diff
@@ -38,14 +41,37 @@
...
+valid_directions = ['n', 'e', 's', 'w']
+while True:
+    # * Prints the current room name
+    print(f'You are currently in {current_player.current_room.name}')
+    # * Prints the current description (the textwrap module might be useful here).
+    print(f'{current_player.current_room.description}')
+    # * Waits for user input and decides what to do.
+    direction = input(f'Where would you like to go? ')
+    if direction in valid_directions:
+        try:
```
Comment (tryingtokeepup) [inline src/adv.py:67]: i love this try/except thing. is everyone using it? is this the new cool kid fad?

File: src/adv.py [hunk 4]
```diff
@@ -38,14 +41,37 @@
...
+            current_player.move(direction)
+            print(f'{current_player.current_room.name}')
+        except:
+            print('YOU DIED!!!')
```
Comment (tryingtokeepup) [inline src/adv.py:71]: brutal

File: src/player.py [hunk 5]
```diff
@@ -1,2 +1,18 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
+
+
+class Player():
+    def __init__(self, name, current_room):
+        self.name = name
+        self.current_room = current_room
+
+    def move(self, direction):
```
Comment (tryingtokeepup) [inline src/player.py:11]: nicely done here!

File: src/room.py [hunk 6]
```diff
@@ -1,2 +1,16 @@
 # Implement a class to hold room information. This should have name and
-# description attributes.
\ No newline at end of file
+# description attributes.
+
+
+class Room:
```
Comment (tryingtokeepup) [inline src/room.py:5]: great start overall :D

**Review by tryingtokeepup — COMMENTED** [review]
Well done well done! I think you hit MVP!

File: src/adv.py (hunk 3 above)
Comment (UnknownMonk) [reply src/adv.py:67]: Yeah man every one is doing it..lol

**Review by tryingtokeepup — APPROVED** [review]
I think everything looks great! And I love the items you put in! Good luck on the sprint today!

**Pull request merged**